simpledotorg/simple-server
# Rewrite district facility downloads to match state downloads

**Story cards:**
- [sc-6277](https://app.shortcut.com/simpledotorg/story/6277/merge-the-state-and-district-level-report-code-into-one-service)
- [sc-6275](https://app.shortcut.com/simpledotorg/story/6275/add-a-facility-size-column-to-the-district-report)

## Because

The state data download uses only the `Repository` for data, and we needed to bring the former district downloads up to speed.

## This addresses

This keeps the reports separate (to avoid tons of conditionals in a single file), but makes the data match up. I also added facility size, as requested by the CVHOs.

## Test instructions

1. Go to the test app
2. Click on "Reports", make sure you're looking at a district level report (one level below state)
3. Click the "Download" button top right
4. Click "Monthly facility data"
